src/tables/colr.rs

```diff
@@ -14,6 +23,139 @@ struct BaseGlyphRecord {
     num_layers: u16,
 }
 
+/// A [ClipBox](https://learn.microsoft.com/en-us/typography/opentype/spec/colr#baseglyphlist-layerlist-and-cliplist).
```

Comment:
We already have a `f32` rect called `BBox` in lib.rs.
How about we rename it to `RectF` and then `ClipBox` will be just a typedef?

Reply:
Sounds good.